### PR TITLE
Game Lab: Animation picker warning

### PR DESCRIPTION
**Pull request by islemaster**
Display this warning message on the Animation Picker dialog when the current user is under the age of 13:

> Warning: Do not upload anything that contains personal information.

pasted_image_at_2016_07_13_14_08

**At the moment this is an unreachable state.** All of gamelab is age-restricted, so any user that is under 13 will get redirected before they can reach this dialog.  This is why I haven't written a UI test for this feature yet.  Once privacy updates are done, we will make gamelab available to students under 13.

To get this working I had to move the `is13Plus`/`is_13_plus` config option in dashboard so that it gets generated on all level views, not just on project views.  This shouldn't break existing behavior around age restrictions and the age prompt dialog.

_Completes [#126192671](https://www.pivotaltracker.com/n/projects/1521049/stories/126192671)_
### Testing
- Manually tested the following matrix with age-restriction redirect disabled: [Game Lab project, Game Lab level] &times; [edit view, readonly view, share view] &times; [over-13 user, under-13 user, signed-out user]
- Passed all apps tests
- Passed projects.feature on local chromedriver, which has to set the `is13Plus` flag in localStorage as part of its steps

### CODE REVIEW

**Comment by Bjvanminnen** [comment]
FYI, paul also has an open PR adding under13 to pageConstants

**Comment by islemaster** [comment]
Oh... maybe I should wait for #9491 then.

File: apps/src/gamelab/AnimationPicker/AnimationPickerBody.jsx [hunk 1]
```diff
@@ -49,3 +56,15 @@ const AnimationPickerBody = React.createClass({
...
+    <span style={warningLabelStyle}>
+      {children}
+    </span>
+);
+WarningLabel.propTypes = {
+  children: React.PropTypes.node
+};
+const warningLabelStyle = {
+  color: color.red
+};
```
Comment (Bjvanminnen) [inline apps/src/gamelab/AnimationPicker/AnimationPickerBody.jsx]: I would personally declare the style before using it. For something as simple as this, you could potentially just do it entirely inline too (I think our styleguide says that's okay if the style is only one field)

File: dashboard/app/helpers/levels_helper.rb [hunk 2]
```diff
@@ -120,6 +120,9 @@ def app_options
 
     set_channel if @level.channel_backed?
 
+    # Always pass user age limit
+    view_options(is_13_plus: current_user && !current_user.under_13?)
```
Comment (Bjvanminnen) [inline dashboard/app/helpers/levels_helper.rb:129]: does this make setting is_13_plus in projects_controller redundant?

**Comment by Bjvanminnen** [comment]
lgtm

File: dashboard/app/helpers/levels_helper.rb (hunk 2 above)
Comment (islemaster) [inline dashboard/app/helpers/levels_helper.rb:129]: Yes, which is why I removed it.

Comment (Bjvanminnen) [inline dashboard/app/helpers/levels_helper.rb:129]: Hah. Good job Brent's brain on missing that, even though it was directly above this. :facepalm:

**Pull request merged**